### DISCUSSION TITLE
Document Upload - allow deleting documents from a Workflow Item when the WF has not been closed yet.

### DISCUSSION THREAD

**Discussion by LukasKM** [openkfw/TruBudget#1113] [Ideas]
Currently, when uploading documents to a WF item, TruBudget does not allow deleting them without cancelling the changes. 
Can you please add a feature next to an uploaded document, e.g. an (x) that would allow to delete it if the WF is not yet saved/closed.

**Comment by nahary** [comment]
> Currently, when uploading documents to a WF item, TruBudget does not allow deleting them without cancelling the changes.
> Can you please add a feature next to an uploaded document, e.g. an (x) that would allow to delete it if the WF is not yet saved/closed.

Hi 
I run the code but I am having problem with the user name and password 
Can you please share with me the username and password 
![Uploading 0F15F047-3EA2-4FC4-BCCF-6F89D74A0485.jpeg…]()

**Comment by nahary** [comment]
![18DA5ABC-7B55-4E5D-8287-D702BE75FF73](https://user-images.githubusercontent.com/30175768/218594867-8d4e0bd2-3b63-4a41-a12d-4ba989584353.jpeg)
